### PR TITLE
Add regression test case

### PR DESCRIPTION
**Pull request by McKnight-42**
#180 

### Description
checks for backwards compatibility in an edge case that database/schema information can be grabbed from default

### Checklist

- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-bigquery next" section.

### CODE REVIEW

File: tests/functional/test_get_columns_incomplete_database.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      - get_cols_in
+"""
+
+class TestIncompleteRelation:
+    @pytest.fixture(scope="class")
+    def properties(self):
+        return {"properties_model_yml.yml": properties__model_yml}
+
+    @pytest.fixture(scope="class")
+    def tests(self):
```
Comment (jtcohen6) [inline tests/functional/test_get_columns_incomplete_database.py:42]: Generic test definitions can go into either the `macros/` directory, or the `tests/generic` subdirectory. It doesn't look like the framework lets us specify the file path as `"generic/get_col_in.sql"`, so macros will do.
```suggestion
    def macros(self):
```
https://docs.getdbt.com/docs/guides/writing-custom-generic-tests

File: tests/functional/test_get_columns_incomplete_database.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        return {"tests__get_col_in.sql": tests__get_cols_in_sql}
+
+    @pytest.fixture(scope="class")
+    def models(self):
+        return { "models__my_model.sql": models__my_model }
```
Comment (jtcohen6) [inline tests/functional/test_get_columns_incomplete_database.py:47]: This model file needs to be named `"my_model.sql"`, so that it matches up with the tests defined on `my_model` in `properties__model_yml`:
```suggestion
        return { "my_model.sql": models__my_model }
```

File: tests/functional/test_get_columns_incomplete_database.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    def test_incomplete_relation(self, project):
+        run_dbt(["run"])
```
Comment (jtcohen6) [inline tests/functional/test_get_columns_incomplete_database.py:50]:
```suggestion
        run_dbt(["run"])
        run_dbt(["test"])
```
Or just:
```suggestion
        run_dbt(["build"])
```

**Review by jtcohen6 — COMMENTED** [review]
Looks good, nearly there! Remember you'll want to include the fix as well when backporting to `1.1.latest`.
https://github.com/dbt-labs/dbt-bigquery/blob/8aaa6ab6da182aa8e831f631feb12af5ec76d9ce/dbt/adapters/bigquery/connections.py#L558-L560

File: tests/functional/test_get_columns_incomplete_database.py (hunk 2 above)
Comment (McKnight-42) [reply tests/functional/test_get_columns_incomplete_database.py:47]: do we need to augment the test name as well in that case? or do we have detection to remove the prefix for tests?

File: tests/functional/test_get_columns_incomplete_database.py (hunk 3 above)
Comment (McKnight-42) [reply tests/functional/test_get_columns_incomplete_database.py]: made suggested changes @jtcohen6

**Comment by McKnight-42** [comment]
the two docs_generate tests that are failing pass locally looking into issue in CI
